siliconcompiler/tools/yosys/syn_asic.py

```diff
@@ -90,6 +84,16 @@ def setup_asic(chip):
                  ",".join(['library', mainlib, 'option', 'var', 'yosys_buffer_output']),
                  step=step, index=index)
 
+    chip.set('tool', tool, 'task', task, 'var', 'map_adders',
+             "true" if chip.valid('library', mainlib, 'option', 'file', 'yosys_addermap') else
+             "false",
+             step=step, index=index, clobber=False)
+    if chip.get('tool', tool, 'task', task, 'var', 'map_adders', step=step, index=index)[0] == \
+       "true":
+        chip.add('tool', tool, 'task', task, 'require',
+                 ",".join(['tool', tool, 'task', task, 'var', option]),
```

Comment:
Is `option` the correct thing to use here? Seems suspect to me, although not 100% sure what the intent is.